snap/hooks/install

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -e
+
+mkdir -p $SNAP_COMMON/libvirt
+cp -r $SNAP/var/snap/multipass/common/libvirt/libvirtd.conf $SNAP_COMMON/libvirt
```

Comment:
I wonder if `install` would be better used here? `-r` doesn't seem necessary?

Reply:
Do you mean the `install` stanza in snapcraft.yaml?  If so, that is install when building the snap.  This is for the actual install of the snap where $SNAP_COMMON will be available.

About `-r`, you're right, that is left over cruft from when I was actually recursively copying directories here, which became unnecessary.  I'll fix that.

Reply:
lol, ok, didn't think of that, but sure:)

Reply:
Ok, uses `install` now.